File: src/main/scala/vm/vagrant/configuration/builder/util/VagrantBuilderException.scala
package vm.vagrant.configuration.builder.util


/**
  * The default Exception for all builder classes.
  *
  * @author oliver.ziegert
  *
  */
@SerialVersionUID(1L)
class VagrantBuilderException(message: String, cause: Throwable) extends RuntimeException (message, cause){
  def this(message: String) {
    this(message, null)
  }

  def this(cause: Throwable) {
    this(null, cause)
  }
}

File: src/main/java/vm/vagrant/configuration/builder/util/VagrantBuilderException.java
package vm.vagrant.configuration.builder.util;

/**
 * The default Exception for all builder classes.
 * @author oliver.ziegert
 *
 */
public class VagrantBuilderException extends RuntimeException {

	private static final long serialVersionUID = 1L;

	public VagrantBuilderException() {
        super();
    }

    public VagrantBuilderException(String message) {
        super(message);
    }

    public VagrantBuilderException(String message, Throwable cause) {
        super(message, cause);
    }

    public VagrantBuilderException(Throwable cause) {
        super(cause);
    }
}
